Kernel/Graphics/GraphicsManagement.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2021, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/NonnullRefPtr.h>
+#include <AK/NonnullRefPtrVector.h>
+#include <AK/Types.h>
+#include <Kernel/Graphics/GraphicsDevice.h>
+#include <Kernel/PCI/Definitions.h>
+
+namespace Kernel {
+
+class GraphicsManagement {
+    AK_MAKE_ETERNAL;
+
+public:
+    static GraphicsManagement& the();
+    static bool is_initialized();
+    bool initialize();
+
+    unsigned current_minor_number() { return m_current_minor_number++; };
```

Comment:
This name makes it sound like it simply returns the `current minor number`, whatever that is.

Let's call it something a bit more accurate, how about `allocate_minor_device_number()`?